tests/sheet/sheet_manipulation_plugin.test.ts

```diff
@@ -200,6 +202,49 @@ describe("Columns", () => {
       expect(model.getters.getColSize(sheetId, 1)).toBe(DEFAULT_CELL_WIDTH);
       expect(model.getters.getActiveSheet().numberOfCols).toBe(2);
     });
+
+    test("On move", () => {
+      model = new Model({
+        sheets: [
+          {
+            id: "sheet1",
+            colNumber: 7,
+            rowNumber: 7,
+            cells: {
+              A1: { content: "A Col" },
+              A2: { content: "1" },
+              A3: { content: "=A2" },
+              A4: { content: "=A2+A3" },
+              B1: { content: "B Col" },
+              B4: { content: "=A2+A3" },
+              C1: { content: "C Col" },
+              C4: { content: "2" },
+              D4: { content: "=A2+A3" },
```

Comment:
isn't 80% of this content useless for your testing purpose ? You only want to test that references are updated right? there should already be tests for moving random stuff in columns